styleguide/STYLE.md

```diff
@@ -0,0 +1,174 @@
+# Styleguide  
+Welcome to the Buildkite styleguide. These are the rules and some useful links for creating new docs. 
+If something isn't included in this guide, see the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/welcome/). 
+
+Table of contents:
+* [Language](#Language)
+* [Style and formatting ](#Style-and-formatting)  
+* [Code and filenames](#Code-and-filenames)
+* [Screenshots](#Screenshots)
+
+## Language  
+This section covers everything related to the language and formatting used in Buildkite documentation (and website, to some extent).
+
+### Dictionary  
+We use American English, so our first-call dictionary is [Merriam Webster](https://www.merriam-webster.com/ ). It is not a single source of truth, just a good starting/reference point.  
+We also use [linter](https://github.com/buildkite/docs#linting) with a file that contains our own [custom list of words](https://github.com/buildkite/docs/blob/master/vale/vocab.txt) that may confuse the spellchecker.
+
+### English flavor 
+Again, we use American English. We're also a highly multi-national team, so here is the list of the most notable [differences between American, British, and Australian English](http://linktranslation.com.au/the-differences-between-american-british-and-australian-english/) Eglish to watch out for.  
+
+### Commas  
+Keep it clean and uncluttered. If the sentence can live without that comma - leave it out.
+
+### Serial commas
+Absolutely yes, yes, and yes.  
+
+### Active/passive voice    
+Generally, use passive voice. It gives a bit more distance from what we're talking about, even if it is a bit longer. There are certainly times for active voice, but it's too nuanced for this guide! [Look here](https://www.aje.com/en/arc/writing-with-active-or-passive-voice/) for some guidance.
+
+### Gender 
+Always use ‘they’, never use ‘he’ or ‘she’. 
+Same, but written by [Microsoft](https://docs.microsoft.com/en-us/style-guide/grammar/nouns-pronouns). 
```

Comment:
This "Same" confused me a little. How about "More info on writing about X in the Microsoft style guide" or something?